video/cloud-client/src/test/java/com/example/video/DetectIT.java

```diff
@@ -121,7 +121,8 @@ public void testTrackObjects() throws Exception {
 
   @Test
   public void testTrackObjectsGcs() throws Exception {
-    VideoAnnotationResults result = TrackObjects.trackObjectsGcs("gs://demomaker/cat.mp4");
+    VideoAnnotationResults result = TrackObjects.trackObjectsGcs(
+            "gs://cloud-samples-data/video/cat.mp4");
```

Comment:
If you have a constant (L44 above), why don't you actually use it?

Reply:
Oh yea, just did a find/replace. 
Will update.